.bufversion

```diff
@@ -0,0 +1 @@
+1.32.2
```

Comment:
Not approving this as our pattern - this would imply to me as a third-party that we have support for this out of the box with our toolchain, which we do not

Reply:
We support this behaviour through [bufisk](https://github.com/bufbuild/bufisk). If not using the `.bufversion` file we can use the env var `BUF_VERSION`. This could be set in the main actions environment or in each workflow file.

Reply:
Bufisk was an experiment that I wrote up in one day and never finished, we never got to stable, and don't advertise. It effectively isn't officially supported. Outside of that, there is no support for `.bufversion`.